docs/INSTALL.md

```
@@ -0,0 +1,145 @@
+    zlib-devel \
+    jansson-devel \
+    hiredis-devel
+```
+
+## Building from Source
+
+### Step 1: Obtain the nginx Source Code
+
+Extract the nginx source code under the OIDC module repository root. The directory structure should look like this:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)